### DISCUSSION TITLE
Show Date and Time

### DISCUSSION THREAD

**Discussion by MattiaFontana997** [primefaces/community#254] [PrimeNG] [answered]
Hello everyone, I'm just approaching the world of primeNg, but looking a bit in the documentation I can't find anything about it.
it is possible to show a date and time in this format (dd/MM/yyyy HH:mm), i have tried with p-Calendar because i want use DatePicker 
Example 
17/10/2023 14:05

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

You can use the time and date format feature for it. Please see the example and read the docs https://primeng.org/calendar#time

**Reply by MattiaFontana997** [reply]
But it doesn't accept an input string on the ngModel
